docs/gettinginvolved/benchmarking.rst

```diff
@@ -0,0 +1,64 @@
+.. _benchmarking:
```

Comment:
I can't figure out how to make a comment on a "no changes" file.  `test_scenarios.py` appears to be missing.